code/modules/projectiles/projectile/magic.dm

```diff
@@ -374,3 +374,12 @@
 	flag = "magic"
 	dismemberment = 50
 	nodamage = 0
+
+/obj/item/projectile/magic/arcane_barrage
+	name = "arcane bolt"
+	icon_state = "arcane_barrage"
+	damage = 20
+	damage_type = BURN
+	nodamage = 0
+	armour_penetration = 100
```

Comment:
@lzimann it's % of armour ignored.
Armour is in percentages, so 60 Melee, you take 40% of incoming damage.
Armour Penetration is subtracted from Armour before every calculation, but can only take Armour as low as 0 (so you'll never GAIN bonus damage from Armour Penetration), so for 100% penetration you will deal 100% of the dame 100% of the time, so people will take 20 damage, regardless of if they're naked or in full riot gear.